src/bugsnag/client/api/Project.ts

```diff
@@ -162,4 +208,475 @@ export class ProjectAPI extends BaseAPI {
       ProjectAPI.buildFields,
     );
   }
+
+  // ============================================================================
+  // Performance API Methods
+  // ============================================================================
+
+  /**
+   * Retrieve the performance score for a specific project.
+   * GET /projects/{project_id}/performance_overview
+   * @param projectId The ID of the project to retrieve the performance score for.
+   * @param releaseStageName The name of the release stage to filter the performance score.
+   * @returns A promise that resolves to the performance score overview.
+   */
+  async getProjectPerformanceScoreOverview(
+    projectId: string,
+    releaseStageName?: string,
+  ): Promise<ApiResponse<ProjectPerformanceScoreOverview>> {
+    const localVarFetchArgs = ProjectsApiFetchParamCreator(
+      this.configuration,
+    ).getProjectPerformanceScoreOverview(projectId, releaseStageName);
+    return await this.requestObject<ProjectPerformanceScoreOverview>(
+      localVarFetchArgs.url,
+      localVarFetchArgs.options,
+    );
+  }
```

Comment:
Performance Score isn't fully live yet - should be removed from here and API class.